README.md

```diff
@@ -193,27 +196,26 @@ released.
 SEMCONV_VERSION=1.8.0 make semconv
 ```
 
-Run it from you bash compatible shell in the root of this repository.
+Run this command in the root of this repository.
 
 ## Styling
 We use [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) for our code linting and standards fixer.  The associated configuration for this standards fixer can be found in the root of the repository [here](https://github.com/open-telemetry/opentelemetry-php/blob/master/.php_cs)
 
-To ensure that your code is stylish, you can run the follwing command:
+To ensure that your code is stylish, you can run the following command:
```

Comment:
Could you please change "is stylish" to "follows our Coding Standard".
This is not about fashion, but about readability.